src/c++/vdo/base/message-stats.h

```diff
@@ -8,6 +8,8 @@
 
 #include "types.h"
 
+int vdo_write_config(struct vdo *vdo, char *buf, unsigned int maxlen);
+
 int vdo_write_stats(struct vdo *vdo, char *buf, unsigned int maxlen);
```

Comment:
We probably don't need a blank line between these signature? Not a big deal either way.

Reply:
Removed